config/application.yml

```diff
@@ -0,0 +1,17 @@
+defaults: &defaults
+  jwt:
+    key: <%= Rails.root.join('config', 'jwt', 'dev.pem') %>
```

Comment:
This value is used also in production env which is not what we want. Dedicated entry with jwt key should be defined for prod env.